can-list.js

```diff
@@ -54,11 +51,11 @@ var List = Map.extend(
 			instances = instances || [];
 			var teardownMapping;
 
-			if (isPromise(instances)) {
+			if (canReflect.isPromise(instances)) {
 				this.replace(instances);
 			} else {
 				teardownMapping = instances.length && mapHelpers.addToMap(instances, this);
-				this.push.apply(this, makeArray(instances || []));
+				this.push.apply(this, canReflect.toArray(instances || []));
```

Comment:
It’d be nice if we changed this to check for `instances`, something like:

```js
instances === undefined ? [] : canReflect.toArray(instances)
```

Reply:
Fixed!